internal/service/searchdeployment/data_source_search_deployment_schema.go

```diff
@@ -9,6 +9,7 @@ import (
 
 func DataSourceSchema(ctx context.Context) schema.Schema {
 	return schema.Schema{
+		MarkdownDescription: "Provides a Search Deployment data source.",
```

Comment:
I thought at the end we would generate both MarkdowDescription and Description cc @AgustinBettati

Reply:
Actually, I added this manually, we could also move this to the template instead. Wondering if there is a way to get the resource description from the open api spec instead. CC @AgustinBettati in case you know how to do this

Reply:
looking at the Hashicorp doc https://developer.hashicorp.com/terraform/plugin/code-generation/openapi-generator#generator-config it does not seem possible. I will open an issue on their repo. In the meantime I will remove this field from the schema and move the description to the template

Reply:
Created this issue https://github.com/hashicorp/terraform-plugin-codegen-openapi/issues/123

Reply:
Yes I agree moving to template will be the best option for now until